### DISCUSSION TITLE
I want to develop a Grammarly-like extension for chrome. is it possible with Quasar?

### DISCUSSION THREAD

**Discussion by its-saeed** [quasarframework/quasar#12630] [CLI - BEX mode]
Hey everyone working with the great quasar framework. I'm going to develop an extension with Quasar and I need to inject some sort of new element inside the active web page. Something like Grammarly.

![grammarly-1](https://user-images.githubusercontent.com/5278422/156306826-f6a00e4f-0721-4cc8-8284-82d544ca4db0.png)

I know that I can inject a new IFrame with Quasar to the page. But what Grammarly does is much more than just adding an iframe. When you hover over a misspelling it displays a dialog containing the correct suggestion. Here is another example of Grammarly:

![grammarly-2](https://user-images.githubusercontent.com/5278422/156307269-9cc3e7b0-d46b-46fc-8f47-ea712a64a612.png)

 Someone told me it's hard to develop such an extension. You are not allowed to use any framework, you can't use any assets.
 
 I saw [this tutorial](https://dev.to/quasar/quasar-and-browser-extension-development-getting-bexy-part-1-39bl). But suppose you want to show a dialog like Grammarly whenever the user clicks the button inside google results.

**Comment by abrehamgezahegn** [comment]


**Reply by its-saeed** [reply]
Hey @abrehamgezahegn great. can we have google meet talk or anything else?
